src/microbe_stage/MicrobeCheatMenu.tscn

```diff
@@ -18,42 +18,49 @@ Resizable = true
 InfiniteCompoundsPath = NodePath("VBoxContainer/InfiniteCompounds")
 GodModePath = NodePath("VBoxContainer/GodMode")
 DisableAIPath = NodePath("VBoxContainer/DisableAI")
+LockTimePath = NodePath("VBoxContainer/LockTime")
 SpeedSliderPath = NodePath("VBoxContainer/Speed/SpeedSlider")
 PlayerDividePath = NodePath("VBoxContainer/PlayerDuplicate")
 SpawnEnemyPath = NodePath("VBoxContainer/SpawnEnemy")
 DespawnAllEntitiesPath = NodePath("VBoxContainer/DespawnAllEntities")
+ManuallySetTimePath = NodePath("VBoxContainer/ManuallySetTime")
+TargetTimePath = NodePath("VBoxContainer/TargetTime/TimeSlider")
 
 [node name="VBoxContainer" type="VBoxContainer" parent="." index="0"]
 margin_right = 177.0
 margin_bottom = 142.0
-__meta__ = {
-"_edit_use_anchors_": false
-}
 
 [node name="InfiniteCompounds" parent="VBoxContainer" index="0" instance=ExtResource( 4 )]
-margin_right = 177.0
+margin_right = 202.0
 margin_bottom = 17.0
 custom_fonts/font = ExtResource( 3 )
 text = "INFINITE_COMPOUNDS"
 
 [node name="GodMode" parent="VBoxContainer" index="1" instance=ExtResource( 4 )]
 margin_top = 21.0
-margin_right = 177.0
+margin_right = 202.0
 margin_bottom = 38.0
 custom_fonts/font = ExtResource( 3 )
 text = "GODMODE"
 
 [node name="DisableAI" parent="VBoxContainer" index="2" instance=ExtResource( 4 )]
 margin_top = 42.0
-margin_right = 177.0
+margin_right = 202.0
 margin_bottom = 59.0
 custom_fonts/font = ExtResource( 3 )
 text = "NO_AI"
 
-[node name="Speed" type="HBoxContainer" parent="VBoxContainer" index="3"]
+[node name="LockTime" parent="VBoxContainer" index="3" instance=ExtResource( 4 )]
```

Comment:
I think the lock time checkbox should be next to the time slider to make the GUI flow more naturally. Grouping related controls in to one place instead of having some unrelated options in-between.